lib/routes/qztc/sjxy/index.ts

```diff
@@ -0,0 +1,120 @@
+import { Data, Route } from '@/types';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import timezone from '@/utils/timezone';
+
+const rootUrl = 'https://www.qztc.edu.cn/sjxy/';
+const host = 'www.qztc.edu.cn';
+
+export const route: Route = {
+    path: '/sjxy/:type',
+    categories: ['university'],
+    example: '/qztc/sjxy/1939',
+    parameters: { type: '分类，见下表' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '泉州师范学院-数学与计算机科学学院 软件学院',
```

Comment:
No need to repeat namespace name
https://github.com/DIYgod/RSSHub/blob/9239d240c16c69e0a96e5e6e31fdb773a426ece4/lib/routes/qztc/namespace.ts#L4 in route name.
```suggestion
    name: '数学与计算机科学学院 软件学院',
```